### PR TITLE
FHAC-729: PerformanceTestTools refactor, Plugins POM cleanup

### PR DESCRIPTION
**Pull request by christophermay07**
* Updated PerformanceTestTools to use standard Mavenized project layout
* Refactored performance test classes for performance and maintainability
* Fixed problems with default performance test configuration files

### CODE REVIEW

**Comment by christophermay07** [comment]
@vdmehta06 @mhpnguyen please review

File: PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/loadtest/DataManager.java [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+    }
+
+    /**
+     * Reads the specified property file as a String
+     *
+     * @param m_sPropertyFile the Property key (ie QUALIFIED_SUBMITTERS)
+     * @return the file's XML read as a string
+     */
+    /*private String getXMLPropertiesOld(String m_sPropertyFile) {
+        String xml = null;
```
Comment (vdmehta06) [inline PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/loadtest/DataManager.java]: Is this commented method is required in future or we can remove it?

File: PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/loadtest/PatientCorrelationBO.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+        try {
+            response = DataManager.getInstance().getCannedPatientCorrelationResponse();
+
+            String targetCommunity = PropertyAccessor.getInstance().getProperty(NhincConstants.GATEWAY_PROPERTY_FILE,
+                CANNED_CORRELATION_COMMUNITY);
+            String patientID = PropertyAccessor.getInstance().getProperty(NhincConstants.GATEWAY_PROPERTY_FILE,
+                CANNED_CORRELATION_PATIENT_ID);
+
+            response.getPRPAIN201310UV02().getControlActProcess().getSubject().get(0)
```
Comment (vdmehta06) [inline PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/loadtest/PatientCorrelationBO.java]: Here we are accessing response object but do we need to check for null before we access 0th obj?

Comment (christophermay07) [inline PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/loadtest/PatientCorrelationBO.java]: If this was real code, yes, but we're guaranteed that either the canned response will have this element, or an exception will be thrown before this point.

File: PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/loadtest/DataManager.java (hunk 1 above)
Comment (christophermay07) [inline PerformanceTestTools/src/main/java/gov/hhs/fha/nhinc/loadtest/DataManager.java]: Whoops, a leftover relic of my initial refactoring. I will remove.

**Comment by mhpnguyen** [comment]
I've review this PR and endorse for merging once all the comments above address

**Comment by vdmehta06** [comment]
I have reviewed changes and good with merge

**Pull request merged**